### PR TITLE
getting data from API to search movies

### PR DESCRIPTION
**Pull request by MirekSobota**

### CODE REVIEW

File: src/common/Navigation/Search/index.js [hunk 1]
```diff
@@ -19,8 +22,11 @@ export const Search = () => {
             peopleMatch ? 'people' : undefined,
             moviesMatch ? 'movies' : undefined,
         );
+        console.log("Testing dispatch value ", target.value);
```
Comment (PawelOwiesek) [inline src/common/Navigation/Search/index.js]: please remove console.log

File: src/features/MovieList/index.js [hunk 2]
```diff
@@ -1,38 +1,45 @@
...
-            />
-          </ItemMoviesList>
-        ))}
-      </MoviesList>
-      <Pagination />
-    </Container>
-  );
+  useEffect(() => {
+    dispatch(fetchMoviesListLoad());
+  }, []);
```
Comment (PawelOwiesek) [inline src/features/MovieList/index.js:17]: dispatch dependency is missing here

File: src/common/Navigation/Search/index.js [hunk 3]
```diff
@@ -15,27 +18,29 @@ export const Search = () => {
     const query = useQueryParameter(searchQueryParamName);
     const replaceQueryParameter = useReplaceQueryParameter();
 
+    const fetchMovies = useCallback(
```
Comment (PawelOwiesek) [inline src/common/Navigation/Search/index.js:21]: Please check these features, something is missing here

File: src/features/movieListSlice.js
Comment (PawelOwiesek) [inline src/features/movieListSlice.js:1]: Everything looks good here

**Review by PawelOwiesek — COMMENTED** [review]
I have a few comments, we will discuss them at the next meeting

**Comment by MirekSobota** [comment]
After consultation with the team, we decided to remove this pull request and use a different strategy for the search.

**Pull request closed without merging**